src/components/footer/index.js

```diff
@@ -3,16 +3,34 @@ import { connect } from 'unistore/preact';
 import config from '../../config';
 import style from './style';
 
+/*
+ *  To update this list, on https://github.com/preactjs/preact/graphs/contributors run:
+ *	$$('.contrib-person [data-hovercard-type="user"]:nth-of-type(2)').map(p=>p.textContent).filter(x => !/-bot$/.test(x)).join(' ')
+ */
+const CONTRIBS = 'developit marvinhagemeister andrewiggins k1r0s cristianbote sventschui JoviDeCroock AlexGalays rpetrich valotas robertknight wardpeet kruczy pmkroeker NekR ForsakenHarmony jviide juicelink billneff79 yaodingyd prateekbh vutran rmacklin impronunciable zouhir scurker SolarLiner mseddon vaneenige lukeed'.split(' ');
+
 @connect( ({ lang, url }) => ({ lang, url }) )
 export default class Footer extends Component {
+	state = {
+		contrib: CONTRIBS[new Date().getMonth()]
+	};
+
+	componentWillReceiveProps({ url }) {
+		if (url!==this.props.url) {
+			this.setState({
+				contrib: CONTRIBS[Math.random()*(CONTRIBS.length-1)|0]
+			});
+		}
+	}
+
 	setLang = e => {
 		this.props.store.setState({ lang: e.target.value });
 	};
 
-	render({ lang, url = location.pathname }) {
+	render({ lang, url = location.pathname }, { contrib }) {
 		let path = url.replace(/\/$/,'') || '/index';
 		if (lang) path = `/lang/${lang}${path}`;
-		let editUrl = `https://github.com/developit/preact-www/tree/master/content${path}.md`;
+		let editUrl = `https://github.com/preactjs/preact-www/tree/master/content${path}.md`;
```

Comment:
Good catch 👍

Reply:
heh - I had noticed the star count API call had started redirecting, saves us the overhead.